packages/core/package.nls.json

```diff
@@ -456,6 +456,10 @@
     "AWS.amazonq.opensettings:": "Open settings",
     "AWS.amazonq.executeBash.run": "Run",
     "AWS.amazonq.executeBash.reject": "Reject",
+    "AWS.amazonq.fsRead.run": "Run",
+    "AWS.amazonq.fsRead.reject": "Reject",
+    "AWS.amazonq.listDirectory.run": "Run",
+    "AWS.amazonq.listDirectory.reject": "Reject",
```

Comment:
nls.json is only necessary if these strings are used in package.json

and for common strings, update `packages/core/src/shared/localizedText.ts`, don't repeat them with different ids. Why would "fsRead" and "listDirectory" have different words for "Run"? Avoid over-abstracting, that is even more costly than not abstracting at all.